### PR TITLE
[PHEE-483] Update upstream with Release version v1.11.0 changes

### PR DESCRIPTION
**Pull request by somanath21**
(https://fynarfin.atlassian.net/browse/PHEE-483)
## Description

* Changes made from the latest release v1.11.0
* Increase GLOBAL_WAIT_TIME_MS
* ImagePullPolicy for integration-test image
* Bullseye image for IAM and MPS (#271) - IAM and MPS docker image upgraded to bullseye image
* Add voucher contact point and IAM
* Fix message-gateway Zeebe-BrokerContactPoint
* Camunda-platform and ES version

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

File: helm/g2p-sandbox-security/Chart.yaml [hunk 1]
```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 description: PaymentHub EE Barebone Edition
 type: application
 name: ph-ee-g2psandbox-security
-version: 0.0.0
-appVersion: 0.0.0
+version: 1.2.0
```
Comment (fynmanoj) [inline helm/g2p-sandbox-security/Chart.yaml:5]: keep this version as 0.0.0

File: helm/g2p-sandbox/values.yaml [hunk 2]
```diff
@@ -369,8 +369,8 @@ ph-ee-engine:
     hostname: bulk-connector.sandbox.mifos.io
     aws:
       region: "<region>"
-      access_key: "<access_key>"
-      secret_key: "<secret_key>"
+      access_key: "aws-access-key"
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:372]: please re-check these values. I think this should be fetched from the secret file

File: helm/ph-ee-engine/values.yaml [hunk 3]
```diff
@@ -2087,27 +2086,16 @@ vouchers:
...
     enabled: true
-
   identity-account-mapper:
-    hostname: "https://identity-mapper.sandbox.mifos.io"
-  operations:
-    hostname: "https://ops-bk.sandbox.mifos.io"
-    endpoints:
-      transfers: "/api/v1/transfers?size=1&page=0"
\ No newline at end of file
+    hostname: "https://ph-ee-identity-mapper:80"
```
Comment (fynmanoj) [inline helm/ph-ee-engine/values.yaml:2101]: operations end point might be needed here, Please check once

File: helm/g2p-sandbox/values.yaml (hunk 2 above)
Comment (somanath21) [reply helm/g2p-sandbox/values.yaml:372]: This is same as the v1.11.0-release chart

File: helm/ph-ee-engine/values.yaml (hunk 3 above)
Comment (somanath21) [reply helm/ph-ee-engine/values.yaml:2101]: This is identity-account-mapper hostname for integration-test

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**